src/adapter/src/subscribe.rs

```diff
@@ -39,6 +41,8 @@ pub struct ActiveSubscribe {
     pub cluster_id: ClusterId,
     /// All `GlobalId`s that the subscribe depend on.
     pub depends_on: BTreeSet<GlobalId>,
```

Comment:
Including this information in `mz_subscribes` would allow people to join on the current names of the dependencies. It doesn't give you the full create SQL, but it gives you some idea of it.

Reply:
Good idea, I'll add that!

Reply:
Was about to leave the exact same comment after reading:

> Subscribes don't have a name, so there is no obvious way for users to map their SQL queries to rows in the table.

Reply:
Added as `referenced_object_ids`.